spec.html

```diff
@@ -33566,8 +33566,8 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( [ _length_ ] )</h1>
-        <p>When the `ArrayBuffer` function is called with optional argument _length_, the following steps are taken:</p>
```

Comment:
Changing the length to zero is inconsistent with other functions, e.g. we could also change DataView.prototype.setInt32's length to zero because that function also "works" when called without any arguments, but the expectation is that setInt32 is called with at least two arguments. And the same applies for ArrayBuffer, where we expect it's called with a single length argument.